.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+Fixed [#issue](link)
+
+## Type of Change
+
+Please delete options that are not relevant.
+
+- [ ] New feature
+- [ ] Bug fix 
+- [ ] This change requires a documentation update
+
+## Proof that it works
+
+Add screenshots or describe the tests that you ran to verify your changes.
+
+## Checklist:
+
+- [ ] I reviewe it myself
```

Comment:
`reviewe` -> `reviewed`

Reply:
ow yes thanks! Actually already fixed that just forgot to push. That happens when you leave things for over the weekend 🙈